### PR TITLE
Fuck this bird

### PR DESCRIPTION
**Pull request by nemvar**
## About The Pull Request

image

## Why It's Good For The Game

FUCKING BIRDBOAT

## Changelog
:cl:
code: Birdboat stuff, the usual. What about it?
/:cl:

### CODE REVIEW

**Comment by wesoda25** [comment]
FUCK BIRDBOAT DUDE

**Comment by Couls** [comment]
:( #47551

**Comment by nemvar** [comment]
wtf did you repath all plastic bottles in that thing? just allow birdboat to eat reagent containers.

File: code/modules/mob/living/simple_animal/hostile/goose.dm [hunk 1]
```diff
@@ -66,17 +66,21 @@
 	. = ..()
 	goosevomit = new
 	goosevomit.Grant(src)
-	RegisterSignal(src, COMSIG_MOVABLE_MOVED, .proc/goosement)
```
Comment (spookydonut) [inline code/modules/mob/living/simple_animal/hostile/goose.dm]: Why not just unregister in goosement() then re-register in a timer callback after some reasonable time

**Comment by TheVekter** [comment]
jesus fucking christ dude

**Comment by MCterra10** [comment]
I'd rather not, nemvar. I have a reputation to uphold

File: code/modules/mob/living/simple_animal/hostile/goose.dm (hunk 1 above)
Comment (nemvar) [reply code/modules/mob/living/simple_animal/hostile/goose.dm]: Why register something to the src obj when I can just use the fucking proc

**Comment by kriskog** [comment]
> 
> 
> why tf did you repath all plastic bottles in that thing? just allow birdboat to eat reagent containers.

That was my kneejerk reaction too, but water bottles contain water which is a food/drinks so it makes sense to just do it finally.

**Pull request closed without merging**